src/sys/resource.rs

```diff
@@ -293,7 +293,9 @@ impl Usage {
         TimeVal::from(self.0.ru_stime)
     }
 
-    /// The resident set size at its peak, in kilobytes.
+    /// /// The resident set size at its peak,
+    #[cfg_attr(apple_targets, doc = " in bytes.")]
+    #[cfg_attr(not(apple_targets), doc = "in kilobytes.")]
```

Comment:
Nix: might be better to have a space here:

```suggestion
    #[cfg_attr(not(apple_targets), doc = " in kilobytes.")]
```